### PR TITLE
add static list of posts solution

### PR DESCRIPTION
**Pull request by opiquem**
[DEMO LINK](https://maksym-bobryk.github.io/react_static-list-of-posts/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,109 +1,30 @@
...
-                href="mailto:Nikita@garfield.biz"
-              >
-                Nikita@garfield.biz
-              </a>
-            </div>
+const allPosts: Post[] = postsFromServer.map(post => ({
+  ...post,
+  user: getUser(post.userId),
+  comments: getComments(post.id),
+}));
```
Comment (Ihor-Karpyn) [inline src/App.tsx:23]: Could we move these functions to a separate file

File: src/App.tsx [hunk 2]
```diff
@@ -1,109 +1,30 @@
...
+);
 
-              <a
-                className="CommentInfo__email"
-                href="mailto:Nikita@garfield.biz"
-              >
-                Nikita@garfield.biz
-              </a>
-            </div>
+const allPosts: Post[] = postsFromServer.map(post => ({
```
Comment (Ihor-Karpyn) [inline src/App.tsx:19]:
```suggestion
const posts: Post[] = postsFromServer.map(post => ({
```

File: src/components/PostInfo/PostInfo.tsx [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+
+        </p>
```
Comment (Ihor-Karpyn) [inline src/components/PostInfo/PostInfo.tsx:29]: Looks like `<p>` and text `posted by` related to UserInfo. Should it be inside `UserInfo`?

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
GJ, almost done

**Review by 33Orange — APPROVED** [review]
Well done

**Pull request closed without merging**